pom.xml

```diff
@@ -106,6 +106,7 @@
     <opencsv.version>2.3</opencsv.version>
     <jcommander.version>1.35</jcommander.version>
     <commons-codec.version>1.10</commons-codec.version>
+    <zstd-jni.version>1.3.8-6</zstd-jni.version>
```

Comment:
Maybe set this to `1.4.0-1` right away.

Reply:
Thanks for the comment @Fokko! [Avro 1.9.0 seems to depend on zstd-jni 1.3.8-6](https://github.com/apache/avro/blob/release-1.9.0/lang/java/pom.xml#L65), so I reconciled them. Should I update it?

Reply:
Good point, maybe leave it as is for now to avoid conflicts

Reply:
https://github.com/apache/avro/pull/549